app/lib/get-subject-location.coffee

```diff
@@ -1,6 +1,7 @@
 READABLE_FORMATS =
   image: ['jpeg', 'png', 'svg+xml', 'gif']
   video: ['mp4']
+  audio: ['mpeg']
```

Comment:
Probably not important, but this doesn't match the MIME type used for the test subject.